app/helpers/local_time_helper.rb

```diff
@@ -67,4 +53,33 @@ def extract_options_and_value(options, value_key = nil)
         [ {}, options ]
       end
     end
+
+    def find_time_format(format)
+      if format.is_a?(Symbol)
+        find_time_format_by_name(format)
+      else
+        [format.presence || LocalTime.default_time_format, nil]
+      end
+    end
+
+    def find_time_format_by_name(format)
+      if format12 = i18n_time_or_date_format(format)
+        format24 = i18n_time_or_date_format("#{format}_24h")
+      elsif fmt = ruby_time_or_date_format(format)
+        format12 = fmt.is_a?(Proc) ? nil : fmt
```

Comment:
Do we need this? I can only see that we're calling this from [here](https://github.com/basecamp/local_time/pull/135/files#diff-d0283343941c298e4f4eeebb501cfb6b01b2380b927ea6e14a7c7a9f771528f6R59) where we've already guarded on sym?